qa/tasks/scripts/ses_disk_fault_injection.sh

```diff
@@ -9,8 +9,12 @@ function health_ok() {
     done
 }
 
-storage_device_name=$(salt $disk_storage_minion cmd.run \
- "pvdisplay | grep -B 1 'VG Name .* ceph' | egrep -v 'ceph-block-dbs|nvme|--' | head -1 | cut -d / -f 3" --output=json | jq -r .[])
+minion_host="$(cut -d '.' -f1 <<< $disk_storage_minion)"
+random_osd=$(ceph osd tree | grep -A 1 $minion_host | grep -o "osd\.".* | awk '{print$1}')
```

Comment:
Same here, please accompany with the comment which string is it transforms to what.